catalog/resource_grants.go

```diff
@@ -159,27 +159,62 @@ var mapping = securableMapping{
 	"table": {
 		"MODIFY": true,
 		"SELECT": true,
+
+		// v1.0
+		"ALL_PRIVILEGES": true,
 	},
 	"view": {
 		"SELECT": true,
 	},
 	"catalog": {
 		"CREATE": true,
 		"USAGE":  true,
+
+		// v1.0
+		"ALL_PRIVILEGES": true,
+		"USE_CATALOG":    true,
+		"CREATE_SCHEMA":  true,
 	},
 	"schema": {
 		"CREATE": true,
 		"USAGE":  true,
+
+		// v1.0
+		"ALL_PRIVILEGES":           true,
+		"USE_SCHEMA":               true,
+		"CREATE_TABLE":             true,
+		"CREATE_VIEW":              true,
+		"CREATE_FUNCTION":          true,
+		"CREATE_MATERIALIZED_VIEW": true,
 	},
 	"storage_credential": {
 		"CREATE_TABLE": true,
 		"READ_FILES":   true,
 		"WRITE_FILES":  true,
+
+		// v1.0
+		"ALL_PRIVILEGES":        true,
+		"CREATE_MOUNT":          true,
+		"CREATE_EXTERNAL_TABLE": true,
 	},
 	"external_location": {
 		"CREATE_TABLE": true,
 		"READ_FILES":   true,
 		"WRITE_FILES":  true,
+
+		// v1.0
+		"ALL_PRIVILEGES":        true,
+		"CREATE_MOUNT":          true,
+		"CREATE_EXTERNAL_TABLE": true,
+	},
+	"metastore": {
+		// v1.0
+		"CREATE_CATALOG":            true,
```

Comment:
These look right to me, but it might be good to have another pair of eyes on them as I haven't been keeping up to date with the metastore level changes.